lib/srv/db/cloud/aws.go

```diff
@@ -68,37 +68,60 @@ func newAWS(ctx context.Context, config awsConfig) (*awsClient, error) {
 	if err := config.Check(); err != nil {
 		return nil, trace.Wrap(err)
 	}
-	meta := config.database.GetAWS()
-	rds, err := config.clients.GetAWSRDSClient(ctx, meta.Region, cloud.WithAssumeRoleFromAWSMeta(meta))
```

Comment:
I changed this because we don't always need to get an RDS client, but technically it wasn't necessary for the elasticache impl